Xamarin.Forms.Xaml.UnitTests/DataTemplateExtension.xaml.cs

```diff
@@ -0,0 +1,33 @@
+﻿using NUnit.Framework;
+
+using Xamarin.Forms.Core.UnitTests;
+
+namespace Xamarin.Forms.Xaml.UnitTests
+{
+	public partial class DataTemplateExtension : ContentPage
+	{
+		public DataTemplateExtension() => InitializeComponent();
+		public DataTemplateExtension(bool useCompiledXaml)
+		{
+			//this stub will be replaced at compile time
+		}
+
+		[TestFixture]
+		class Tests
+		{
+			[SetUp]
+			public void Setup() => Device.PlatformServices = new MockPlatformServices();
+			[TearDown] public void TearDown() => Device.PlatformServices = null;
```

Comment:
It might be better to be consistent with attributes on a separate line.

Reply:
I agree for consistency. this is boilerplate, and i like the boilerplate to
1. take as few lines as possible
2. be of a recognizable shape

Reply:
fixed